readthedocs/settings/base.py

```diff
@@ -818,6 +818,8 @@ def DOCKER_LIMITS(self):
         r'docs\.python\.org',
         r'docs\.scipy\.org',
         r'docs\.sympy\.org',
+        r'www.sphinx-doc.org',
+        r'numpy\.org',
```

Comment:
This is likely something that should be defined in the DB, or somewhere else that doesn't require a deploy to modify. It's definitely something we're going to need to iterate on pretty frequently I'm guessing, and perhaps even support customization at a project or organization level.

Reply:
I agree. We do have an issue opened for Django settings in the DB at https://github.com/readthedocs/readthedocs-ops/issues/1004 but I don't think that will progress too much. If that's the case, we can create a simple `EmbedDomain` model that we can relate to a Project and/or Organization as well.

Reply:
Opened https://github.com/readthedocs/readthedocs.org/issues/8530 to continue the conversation there.